test/rpt_cmp.py

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
at least add a comment about why this complicated script exists

Reply:
This script is used to compare to other reports from other tools, I was advised to remove it and move it to the private repo.